src/methods.ts

```diff
@@ -254,6 +347,11 @@ export class Method extends AbstractMethodField {
    */
   private code: any;
 
+  private numOpcodeExecs = 0;
+  private jitThreshold = 0;
+  private compiledFunctions: Function[] = [];
```

Comment:
I've actually experimented with a DoppioJVM JIT before. Caching the basic block source lets you progressively JIT more of the function, if you so choose. For example, if a basic block has not run, it still has `invokestatic` and friends instead of `invokestatic_fast`. You may want to re-JIT those basic blocks if they suddenly become hot, since the conversion happens on their first run.

If you are not doing that, then there's no reason to store the source.